api/src/auth.js

```diff
@@ -151,7 +151,7 @@ module.exports = {
 
     let username, password;
     try {
-      [username, password] = new Buffer(authHeader.split(' ')[1], 'base64').toString().split(':');
+      [username, password] = Buffer.from(authHeader.split(' ')[1], 'base64').toString().split(':');
```

Comment:
Can you explain all the changes to `Buffer`? I'm not going to comment on every one, but there are lots of these changes through this PR.

They are also not consistent. For example:
 - `new Buffer(str, type)` to `Buffer.from(str, type)`
 - `Buffer.from(str, type) -> Buffer.from(str).toString(type)`

If we needed to do the second one why do we do the first one?

Reply:
First one is `decoding` and the second one is `encoding` 
Example: In the event there is no `logo` in the `branding doc` encode the logo image on the app to `base64` and render it `Buffer.from(logo).toString('base64');`